cetmix_tower_server/tests/common.py

```diff
@@ -15,6 +15,11 @@ def setUp(self, *args, **kwargs):
         # Cetmix Tower helper model
         self.CetmixTower = self.env["cetmix.tower"]
 
+        # Tags
+        self.Tag = self.env["cx.tower.tag"]
```

Comment:
Здесь без этой переменной мы будем получать ошибку:

```python
"/your/path/cetmix/cetmix_tower/custom_addons/cetmix_tower_server/tests/common.py", line 19, in setUp
    self.tag_test_staging = self.Tag.create({"name": "Test Staging"})
AttributeError: 'TestTowerVariable' object has no attribute 'Tag'
```

Reply:
Could you elaborate on this please, I don't understand what you mean.

Reply:
Previously, `self.Tag` was just for beauty, because I was creating tags directly via `self.env[‘cx.tower.tag’].create()` anyway. It turned out to be a useless variable.

Now I really use it, I create tags through `self.Tag.create()`. The code became simpler and more logical, without unnecessary repetitions. 

And if I remove it, I will have to create tags through a direct call to `self.env[‘cx.tower.tag’].create()`, which will take more space. And now we have an implementation like in `[Server](https://github.com/cetmix/cetmix-tower/blob/a44e50b7b0f0614d2f89ecff514f60246ad9ae74/cetmix_tower_server/tests/common.py#L36)`.